src/App.js

```diff
@@ -1,53 +1,86 @@
 import React, { Component } from 'react';
-import Navbar from './components/Navbar'
-import MovieDetails from './components/MovieDetails'
-import MoviesContainer from './components/MoviesContainer'
+import Navbar from './components/Navbar';
+import MovieDetails from './components/MovieDetails';
+import MoviesContainer from './components/MoviesContainer';
 import './App.css';
 import movieDetails from './data/movieDetails';
-import moviesData from './data/movieData';
+import ErrorMessage from './components/ErrorMessage';
 
 class App extends Component{
   constructor() {
     super();
     this.state = {
       movieSelected: 0,
       movieDetails: movieDetails,
-      movies: []
+      movies: [],
+      errorMessage: ''
     }
   }
 
+  componentDidMount = () => {
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+    .then(response => {
+      if(response.status === 200) {
+        return response.json()
+      }
+      throw new Error(this.getErrorMessage(response.status))})
+    .then(data => this.setState({movies: data.movies}))
+    .catch(error => {
+      this.setState({errorMessage: error.message})
+    })
+  }
+
   handleClick = (id) => {
-    // console.log(id)
-    this.setState({
-      movieSelected: id
+    {!!id ? fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
```

Comment:
Not sure the `!!` is needed here, but I might be missing something. Why couldn't `id ? fetch...` work?